lasagne/layers/prelu.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np
+import theano.tensor as T
+
+from .. import init
+from .. import nonlinearities
+
+from .base import Layer
+
+__all__ = [
+    "PReLULayer"
+]
+
+class PReLULayer(Layer):
+    """
+    A layer used to implement Parametric Rectified Linear Units (PReLU).
+    This layer simply applies the parametric nonlinearity to the previous
+    layer's output. 
+
+    Parameters
+    -----------
+    incoming : a :class:`Layer` instance or a tuple
+        The layer feeding into this layer, or the expected input shape.
+
+    alpha : double or None
+        The initial "leakiness" of the rectifier which is then learned.
+        If argument not provided, initial leakiness will be set to 0.25.
+
+    unique_chan_param : boolean or None
+        When true, this argument creates a vector of alphas rather than a
+        single theano scalar. Each scalar is applied to a separate channel
+        of the previous layer's output. Should only be used if the 
+        previous layer is a convolution. If argument not provided, it will
+        be set to False.
+
+    Usage
```

Comment:
I believe this should be 'Examples' instead of 'Usage' for sphinx to pick it up.